### PR TITLE
Empty state updates.mdx

### PR DESCRIPTION
**Pull request by mma1504**
Made significant updates to the empty state documentation to provide content design guidance.

**Contributing to Twilio**

> All third-party contributors acknowledge that any contributions they provide will be made under the same open-source license that the open-source project is provided under.

- [ ] I acknowledge that all my contributions will be made under the project's license.

### CODE REVIEW

**Review by nkrantz — APPROVED** [review]

File: packages/paste-website/src/pages/patterns/empty-state/index.mdx [hunk 1]
```diff
@@ -210,109 +192,193 @@ Open the sandbox in a new tab and resize the browser for an example of responsiv
...
+      </Tr>
+      <Tr verticalAlign="top">
+        <Td scope="row">User doesn’t have access or permissions</Td>
+        <Td><b>No [thing] yet</b> <br/> You'll see [thing] here when [something happens].</Td>
+        <Td>Optional</Td>
+      </Tr>
+   <Tr verticalAlign="top">
+        <Td scope="row">404 page</Td>
+        <Td><b>Page not found</b> <br/> he page you're looking for either doesn't exist or was moved.</Td>
+        <Td>Go back</Td>
```
Comment (TheSisb) [inline packages/paste-website/src/pages/patterns/empty-state/index.mdx:330]: thought (non-blocking): The CTA column header is a little confusing to me. In the previous table the entire column was marked as "Optional", which made me thing the CTA column header should be renamed to "CTA required?". But in this table it jumps from `Optional` to `Go back`, which makes me do a double-take at whether the CTA is supposed to say "Optional".  

Maybe a suggestion instead is to provide an example CTA but append it with `(optional)`.  Ex:
```
<Td><b>No [thing] yet</b> <br/> You'll see [thing] here when [something happens].</Td>
<Td>Return to the [thing] page (optional)</Td>
```

Comment (serifluous) [reply packages/paste-website/src/pages/patterns/empty-state/index.mdx:330]: +1

File: packages/paste-website/src/pages/patterns/empty-state/index.mdx [hunk 2]
```diff
@@ -210,109 +192,193 @@ Open the sandbox in a new tab and resize the browser for an example of responsiv
...
+        <Td>Optional</Td>
+      </Tr>
+      <Tr verticalAlign="top">
+        <Td scope="row">User doesn’t have access or permissions</Td>
+        <Td><b>No [thing] yet</b> <br/> You'll see [thing] here when [something happens].</Td>
+        <Td>Optional</Td>
+      </Tr>
+   <Tr verticalAlign="top">
+        <Td scope="row">404 page</Td>
+        <Td><b>Page not found</b> <br/> he page you're looking for either doesn't exist or was moved.</Td>
```
Comment (serifluous) [inline packages/paste-website/src/pages/patterns/empty-state/index.mdx:329]:
```suggestion
        <Td><b>Page not found</b> <br/> The page you're looking for either doesn't exist or was moved.</Td>
```

File: packages/paste-website/src/pages/patterns/empty-state/index.mdx [hunk 3]
```diff
@@ -210,155 +194,258 @@ Open the sandbox in a new tab and resize the browser for an example of responsiv
...
+    style={{width: '100%', height: '350px', border: '0', borderRadius: '4px', overflow: 'hidden'}}
+    title="Empty state requiring a customer to wait"
+    allow="accelerometer; ambient-light-sensor; camera; encrypted-media; geolocation; gyroscope; hid; microphone; midi; payment; usb; vr; xr-spatial-tracking"
+    sandbox="allow-forms allow-modals allow-popups allow-presentation allow-same-origin allow-scripts"
+  ></iframe>
+</Box>
+
+### Error states
+
+Error empty states can result from problems like a system issue or access issue. The content should explain what went wrong and tell the user how to fix the issue. Consider including an error icon to complement the content. If the error empty state occurs in a small component (a card, for example), prioritize communicating how to fix the problem. For error message guidance, review the [error state pattern] (patterns/error-state).
```
Comment (serifluous) [inline packages/paste-website/src/pages/patterns/empty-state/index.mdx:336]:
```suggestion
Error empty states can result from problems like a system issue or access issue. The content should explain what went wrong and tell the user how to fix the issue. Consider including an error icon to complement the content. If the error empty state occurs in a small component (a card, for example), prioritize communicating how to fix the problem. For error message guidance, review the [error state pattern](/patterns/error-state).
```

**Review by SiTaggart — APPROVED** [review]

**Review by serifluous — APPROVED** [review]

**Pull request merged**